Momento/HeaderInterceptor.cs

```diff
@@ -2,11 +2,14 @@
 using Grpc.Core;
 using Grpc.Core.Interceptors;
 using System.Collections.Generic;
+using System.Linq;
 
 namespace MomentoSdk
 {
     class Header
     {
+        private const string Agent = "Agent";
```

Comment:
I dont think we should add this in each file. Define once and reference in each class.

e.g. drop the constant private here
```
const string AGENT_KEY = "Agent";
const string AUTHORIZATION_KEY = "Authorization";
```

And then in DataGrpcManager reference in `Headers.AGENT_KEY`

Reply:
And just one note, you probably should looking up naming conventions for C# for constants `AGENT_KEY` is what is done in Java. So, check the appropriate C# guideline.

Reply:
It seems like `FullProductName` is its naming convention for constants.
https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/keywords/const